### ISSUE TITLE
fcntl on windows

### ISSUE THREAD

**Issue by h0nda** [dmlc/tvm#1007]
```
c:\>python -c "import tvm"

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "c:\tvm\python\tvm\__init__.py", line 14, in <module>
    from . import module
  File "c:\tvm\python\tvm\module.py", line 9, in <module>
    from .contrib import cc as _cc, tar as _tar, util as _util
  File "c:\tvm\python\tvm\contrib\cc.py", line 9, in <module>
    from .util import tempdir
  File "c:\tvm\python\tvm\contrib\util.py", line 5, in <module>
    import fcntl
ModuleNotFoundError: No module named 'fcntl'
```

**Comment by hongdayu** [comment]
how to solve it??

**Comment by hongdayu** [comment]
C:\Users\yuhongda\tvm>python
Python 3.5.4 (v3.5.4:3f56838, Aug  8 2017, 02:17:05) [MSC v.1900 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tvm
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Users\yuhongda\AppData\Roaming\Python\Python35\site-packages\tvm-0.2.0-py3.5-win-amd64.egg\tvm\__init__.py", line 14, in <module>
    from . import module
  File "C:\Users\yuhongda\AppData\Roaming\Python\Python35\site-packages\tvm-0.2.0-py3.5-win-amd64.egg\tvm\module.py", line 9, in <module>
    from .contrib import cc as _cc, tar as _tar, util as _util
  File "C:\Users\yuhongda\AppData\Roaming\Python\Python35\site-packages\tvm-0.2.0-py3.5-win-amd64.egg\tvm\contrib\cc.py", line 9, in <module>
    from .util import tempdir
  File "C:\Users\yuhongda\AppData\Roaming\Python\Python35\site-packages\tvm-0.2.0-py3.5-win-amd64.egg\tvm\contrib\util.py", line 5, in <module>
    import fcntl
ImportError: No module named 'fcntl'

**Comment by hongdayu** [comment]
Nnvm also has this problem when build on windows

**Comment by tqchen** [comment]
should be fixed by #1009
